pkgs/applications/misc/blender/wrapper.nix

```diff
@@ -2,7 +2,7 @@
 , lib
 , blender
 , makeWrapper
-, python3Packages
+, pythonPackages
```

Comment:
Define the default value of `pythonPackages` as `blender.pythonPackages`, that way you can omit the pythonPackages parameter where you are calling this function.

Reply:
`pythonPackages ? blender.pythonPackages`

Just check if it gets the default values right.